src/dev/riscv/LupV.py

```diff
@@ -0,0 +1,182 @@
+from m5.objects.Platform import Platform
+from m5.objects.PMAChecker import PMAChecker
+from m5.objects.Clint import Clint
+from m5.objects.LupioRTC import LupioRTC
+from m5.objects.Plic import Plic
+from m5.objects.RTC import RiscvRTC
+from m5.objects.Uart import Uart8250
+from m5.objects.Terminal import Terminal
+from m5.params import *
+from m5.proxy import *
+from m5.util.fdthelper import *
+
+class LupV(Platform):
+    """LupV Platform
```

Comment:
Maybe a comment here saying that you copied this from HiFive.py and that you'll be updating it over a number of commits.